sessions/nextjs-dynamic-routes/lotr-app-dynamic-routes/README.md

```diff
@@ -0,0 +1,52 @@
+# Next.js Dynamic Routes: Lord of the Rings App Dynamic Routes
+
+Currently, the overview about the Lord of the Rings volumes is done with static routing. You are going to refactor the code with dynamic routes as well as links. Finally, you will create a button to navigate to a random volume.
+
+## Task
+
+### Refactor: Dynamic Routes
+
+Refactor the static routes of the volume details so that they use dynamic routes instead.
+
+You can use the following hints as guideline:
+
+- Create a `[slug].js` file in the `volumes` folder.
+- Inside of `[slug].js` create a `VolumeDetail` component which returns the same elements as the static volume did, but change the content to be now based on the `slug` from the `router.query` object.
```

Comment:
```suggestion
- Inside of `[slug].js` recreate the `VolumeDetail` component which returns the same elements as the static version of the component. Adjust the code so that the content is now based around the `slug` from the `router.query` object.
```
Maybe?